src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsCard/attributes/DisplayAttributeInputs.tsx

```diff
@@ -88,6 +91,19 @@ const DisplayAttributeInputs = ({
             label="Integer"
           />
         );
+
+      case "Quantity":
+        return (
+          <QuantityInput
+            quantity={{}}
+            onQuantityChange={(val) => {
+              if (val.unit && val.value) {
+                setAttributeValue(new CQL.Quantity(val.value, val.unit));
+              }
```

Comment:
Quantity object creation should be done in the Quantity component. No need to check here again.

Reply:
Done